<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralFormClass</class>
 <widget class="QWidget" name="GeneralFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>334</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>283</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GeneralForm</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #F1F1F1;</string>
  </property>
  <widget class="QCheckBox" name="checkBox_hidechar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>hide char</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_lockmove">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>lock move</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_autojoin">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>auto join</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_autostack">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>auto stack</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_console">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>console</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_autodropmeat">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>auto dpmeat</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_locktime">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>lock time</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_closeeffect">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>close effect</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_autoeatbean">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>auto eat bean</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_autoanswer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>370</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>auto answer</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_knpc">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>knpc</string>
   </property>
  </widget>
  <widget class="ComboBox" name="comboBox_locktime">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>66</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_autostartscript">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>auto start script</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_mute">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>mute</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_fastwalk">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>fast walk</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_autodrop">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>auto drop</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_autorestart">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>auto restart</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_lockimage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>390</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>lock image</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_showexp">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>show exp</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_autoswitch">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>auto switch</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_fastbattle">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>fast battle</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_lockescape">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>lock escape</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_autoescape">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>auto escape</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_lockattck">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>lock attack</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_autowalk">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>auto walk</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_battletimeextend">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>battle time</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_autocatch">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>auto catch</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_falldownescape">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>fall escape</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_battleautoeo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>autoeo</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_autobattle">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>auto battle</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_fastautowalk">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>fs auto walk</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>5</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>boost</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_speedboost">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>5</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="maximum">
    <number>12</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_main">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>65</y>
     <width>23</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>main</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_char">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>113</y>
     <width>23</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>23</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>pos</string>
   </property>
  </widget>
  <widget class="PushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>204</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
  <widget class="ComboBox" name="comboBox_position">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>113</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="PushButton" name="pushButton_setting">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>5</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>set</string>
   </property>
  </widget>
  <widget class="ComboBox" name="comboBox_paths">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>181</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_autologin">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>136</y>
     <width>67</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>autologin</string>
   </property>
  </widget>
  <widget class="PushButton" name="pushButton_addpath">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>181</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="ComboBox" name="comboBox_server">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>65</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="label_vice">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>90</y>
     <width>23</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>23</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>vice</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_hidewindow">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>207</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>hide win</string>
   </property>
  </widget>
  <widget class="ComboBox" name="comboBox_subserver">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>90</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_autoreconnect">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>136</y>
     <width>68</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>reconnect</string>
   </property>
  </widget>
  <widget class="ComboBox" name="comboBox_setting">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>5</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="PushButton" name="pushButton_logout">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>155</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>logout</string>
   </property>
  </widget>
  <widget class="ComboBox" name="comboBox_serverlist">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="PushButton" name="pushButton_logback">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>155</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>logback</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_switcher_card">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>288</y>
     <width>35</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CD</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_switcher_trade">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>288</y>
     <width>34</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TR</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_switcher_team">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>288</y>
     <width>35</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TM</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_switcher_job">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>305</y>
     <width>35</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>JB</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_switcher_pk">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>288</y>
     <width>35</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PK</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_switcher_group">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>305</y>
     <width>35</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>GM</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_switcher_family">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>305</y>
     <width>35</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>FM</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_switcher_world">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>305</y>
     <width>34</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>WD</string>
   </property>
  </widget>
  <widget class="PushButton" name="pushButton_dock">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>236</y>
     <width>32</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>dock</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="PushButton" name="pushButton_joingroup">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>236</y>
     <width>32</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>join</string>
   </property>
  </widget>
  <widget class="PushButton" name="pushButton_savesettings">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>236</y>
     <width>32</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="PushButton" name="pushButton_clear">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>236</y>
     <width>32</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </widget>
  <widget class="PushButton" name="pushButton_leavegroup">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>260</y>
     <width>32</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>leave</string>
   </property>
  </widget>
  <widget class="PushButton" name="pushButton_afksetting">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>260</y>
     <width>32</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>afk</string>
   </property>
  </widget>
  <widget class="PushButton" name="pushButton_eo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>32</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>eo</string>
   </property>
  </widget>
  <widget class="PushButton" name="pushButton_loadsettings">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>260</y>
     <width>32</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>load</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">combobox.h</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header location="global">pushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
